### PR TITLE
Updated the Themes Admin View to better support strings localization

### PR DESCRIPTION
**Pull request by PiemP**

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Themes/Views/Admin/Index.cshtml [hunk 1]
```diff
@@ -59,7 +59,7 @@ else
                         }
                         else
                         {
-                            @T["This is the default {0} theme", themeEntry.IsAdmin ? "Admin" : "Site"]
+                            @T["This is the default {0} theme", themeEntry.IsAdmin ? @T["Admin"] : @T["Site"]]
```
Comment (sebastienros) [inline src/OrchardCore.Modules/OrchardCore.Themes/Views/Admin/Index.cshtml]: If a parameter needs to be translated, then it should use the `IStringLocalizer` instead.
Another simpler option is to use two strings: 
```suggestion
                            if (themeEntry.IsAdmin)
                            {
                                @T["This is the default Admin theme"]
                            }
                            else
                            {
                                @T["This is the default Site theme"]
                            }
```

**Comment by hishamco** [comment]
LGTM

**Comment by hishamco** [comment]
@PiemP could you resolve the conflict

**Comment by agriffard** [comment]
In a recent change, we chose to replace `default` by `current` theme.
In your change, you are adding `default` again.

**Comment by PiemP** [comment]
updated the label to use the `current` term instead of `default` term

**Comment by agriffard** [comment]
With is there a change in appveyor.yml ?

**Pull request merged**